autoload/flog.vim

```diff
@@ -1012,6 +1012,190 @@ endfunction
 
 " }}}
 
+" Quick navigate operations {{{
+
+" if the star symbol ever changes, this will remain
+" the backwards compatible way of going to the last
+" char that is a part of the graph drawing
+fu! flog#to_commit_start() abort
+  execute "normal! 0f*"
```

Comment:
I believe in some special cases other characters will be used for the starting character (avoiding this issue is why this functionality doesn't currently exist), and also sometimes a commit will currently start with `...` since fugitive doesn't currently handle the null commit area when skipping commits (which I think this will fail silently with).

I believe the special characters are `+` and `-`, and maybe `<`/`>`. I have to check the git manual, but I don't have a chance right now.

Reply:
Ok; but since `0f*` just leaves the cursor at the beginning of the line if there is no star, I'm not too concerned about this for now.

Reply:
Fair enough. It could jump to a character in the commit log but I'm not super worried about that. I'll make an issue to deal with it later.

Reply:
I've removed this. It doesn't make sense now that the user can customize the log graph generation command.